siriuspy/siriuspy/devices/bpm.py

```diff
@@ -803,6 +845,15 @@ def cmd_sync_monit(self):
         self.monit_sync_enbl = 0
         return self._wait('Monit1TagEn-Sts', 0)
 
+    def _get_postmorten_prop(self, prop):
```

Comment:
wouldn't be better to use `_get_propname` or something that do not use "postmorten" in name?

Reply:
right, done!